### ISSUE TITLE
Use of an input file instead of the editor 

### ISSUE THREAD

**Issue by mlerman** [go-jira/jira#300]
Very simple question. Is it possible to use a text file instead of being prompted to use the editor?
For example I would need an option `--file="path\to\input_file" that replaces --editor="notepad++.exe"`
I can this way prepare the input file prior to launching the jira command, and get through this step without user interaction.

**Comment by coryb** [comment]
You can use the `--noedit` option to prevent the editor from opening.  I think the template need to be in .jira.d/templates now, but you can create a fully filled in template in like at `.jira.d/templates/custom-issue` then run `jira create -t custom-issue --noedit`

**Comment by mlerman** [comment]
Thank you coryb. I never succeeded to use the templates.
I did export the templates with the command `jira.exe export-templates` and it exports a bunch of files in C:\Users\me\.jira.d\templates\
But I still don't know how to export a custom issue, and reference it, with the create command, as you mentioned. 
I have checked here: https://easesolutions.atlassian.net/wiki/spaces/REQ4J/pages/64920002/Configuring+Custom+Export+Templates 
It seems that Jira administrators can do that, but not plain user.

When I use the --noedit option and override the required fields. It is still not taking all of them :(

My proposed option --file is simple. Instead of using the editor output file, just use the referenced file. 
I am used to this method and I am sure many others feel the same way. This --editor method is obscure and unusual.

**Comment by coryb** [comment]
Yeah, I agree, the editor hack is obscure.  This is an open source project and we will review pull requests  😄 So feel free to add a `--file` option to `create` and/or update the `jira request` command to do similar.  I have very little time these days so I am unlikely to implement this myself.

**Comment by mlerman** [comment]
@coryb thank you. I have submitted the pull request.
